src/main/java/mate/controller/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,21 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        driverService.delete(Long.parseLong(req.getParameter("id")));
```

Comment:
what is the difference between `Long.parseLong` and `Long.valueOf`
check all controllers